src/main/java/org/launchcode/liftoffrecipetracker/Models/User.java

```diff
@@ -0,0 +1,56 @@
+package org.launchcode.liftoffrecipetracker.Models;
+
+import javax.persistence.Entity;
+import java.util.ArrayList;
+
+@Entity
+public class User extends org.launchcode.liftoffrecipetracker.models.AbstractEntity {
+
+    private String username;
+    private String password;
+    private String email;
+    private ArrayList recipe = new ArrayList();
+
+    public User(){
+
+    }
+
+    public User(String username, String password, String email, ArrayList recipe) {
+        this.username = username;
+        this.password = password;
+        this.email = email;
+       this.recipe = recipe;
```

Comment:
Super minor point: can you add a space at the beginning of this.recipe so it is in line with the other variables!